documentation/3.3/content/userguide/aks/_index.md

```diff
@@ -296,6 +297,31 @@ If you want to use this template to automate the deployment, download it by sele
 
 Click **Create** to create this offer. This process may take 30 to 60 minutes.
 
+#### Template outputs
+
+After clicking **Create** to create this offer, you will go to the **Deployment is in progress** page. When the deploymet is completed, the page shows **Your deployment is complete**. In the left panel, select **Outputs**. These are the outputs from the deployment.  The following table is a reference guide to the deployment outputs.
+
+| Field | Description |
+|-------|-------------|
+| `aksClusterName` | Name of your AKS cluster that is running the WLS cluster. {{< line_break >}}Sample value: `wlsonaksiyiql2i2o2u2i`. |
+| `adminConsoleInternalUrl` | The fully qualified, private link to the Administration Console portal. You can only access it inside AKS cluster. {{< line_break >}}Sample value: `http://sample-domain1-admin-server.sample-domain1-ns.svc.cluster.local:7001/console`. |
+| `adminConsoleExternalUrl` | This output is not always present:{{< line_break >}}You must configure [Networking](#networking) to enable Azure Load Balancer service or Azure Application Gateway Ingress Controller for the Administration Console.{{< line_break >}} {{< line_break >}}This is a fully qualified, public link to the Administration Console portal. You can access it from the public Internet. {{< line_break >}}Sample value: `http://wlsgw202208-wlsd-aks-2793762585-337-domain1.eastus.cloudapp.azure.com/console`. |
+| `adminConsoleExternalSecuredUrl` | This output is not always present:{{< line_break >}}1. You must configure [Networking](#networking) to enable Azure Load Balancer service or Azure Application Gateway Ingress Controller for the Administration Console.{{< line_break >}}2. You must configure a custom DNS name by filling out [DNS Configuration](#dns-configuration).{{< line_break >}}3. The TLS/SSL certificate used is configured by filling out [TLS/SSL configuration](#tlsssl-configuration).{{< line_break >}}{{< line_break >}}This is a fully qualified, secure, public link to the Administration Console portal. You can access it from the public Internet. {{< line_break >}}Sample value: `https://contoso.com/console`. |
```

Comment:
to enable Azure Load Balancer service -> to enable the Azure Load Balancer service (please make this correction wherever this text appears)